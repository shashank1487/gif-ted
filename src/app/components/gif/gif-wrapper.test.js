import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import GifWrapper from "./gif-wrapper";

configure({ adapter: new Adapter() });
describe("The gif wrapper component", () => {
  let app;

  beforeEach(() => {
    app = shallow(
      <GifWrapper
        src="https://media3.giphy.com/media/pPhyAv5t9V8djyRFJH/giphy.gif"
        still="https://media1.giphy.com/media/pPhyAv5t9V8djyRFJH/480w_s.jpg"
      />
    );
  });

  it("should render correctly", () => {
    expect(app).toMatchSnapshot();
  });

  it("should render GifPlayer component", () => {
    expect(app.find("GifPlayerContainer").exists()).toBeTruthy();
  });
});
