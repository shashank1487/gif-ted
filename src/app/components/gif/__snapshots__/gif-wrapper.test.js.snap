// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The gif wrapper component should render correctly 1`] = `ShallowWrapper {}`;
